source/fundamentals/connection/network-compression.txt

```diff
@@ -0,0 +1,98 @@
+.. _compression:
+.. _network-compression:
+
+===================
+Network Compression
+===================
+
+The {+driver-long+} provides a connection option to compress messages,
+This reduces the amount of data passed over the network between MongoDB
+and your application.
+
+The driver supports the following algorithms:
+
+1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.4 and later.
+#. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
```

Comment:
is there a reason Zlib isn't included in the code examples?

Reply:
No, I just randomly selected 2 of the 3 to illustrate how to include them. But I can include all three in both snippets